### ISSUE TITLE
Rephrase ALT to TEXT 

### ISSUE THREAD

**Issue by chagai95** [mastodon/mastodon#31441]
### Steps to reproduce the problem

On the image it says ALT

### Expected behaviour

it should be something understandable for users 

### Actual behaviour

ALT

### Mastodon instance

mastodon.social

### Mastodon version

v4.1.2

### Browser name and version

Chrome

### Operating system

Android

**Comment by rasos** [comment]
Agree, we programmers now the ALT html tag, but a normal users has no idea what this abbreviation means . In German alt means old. 

MAybe we should not show ALT or TEXT at all, but simply show the alt text as discussed here https://github.com/mastodon/mastodon/issues/31642 with PR https://github.com/mastodon/mastodon/pull/31662 but not only in notifications.

**Comment by Ember-ruby** [comment]
'TEXT' would probably be mistaken as some sort of classification for the image
showing the alt text at all times may be an issue for longer alt text

ideally i think the label would be a button that shows the full alt text in a pop-up when pressed

**Comment by fccfc5d245c** [comment]
Good idea, but wouldn't it be better to use the word "DESCRIPTION"? "TEXT" could be misinterpreted in many unpredictable ways, "description" in self-explanatory.

Or any shorter synonym, I can't think of one right now.
